### PR TITLE
added composer

### PR DESCRIPTION
**Pull request by bart-j**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "pleio/friend_request",
```
Comment (jeabakker) [inline composer.json]: this should be coldtrick/friend_request

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+    "type": "elgg-plugin",
+    "require": {
+        "composer/installers": "~1.0"
```
Comment (jeabakker) [inline composer.json]: this should be version ^1.0.8

**Comment by jeabakker** [comment]
please check out https://github.com/ColdTrick/fontawesome/blob/master/composer.json for a version we could implement in all our plugins

**Comment by bart-j** [comment]
ok, will use this template for all my pull requests!

**Comment by jeabakker** [comment]
please also don't forget to push the composer.lck and .gitignore

**Comment by bart-j** [comment]
ok will do that!

*Bart Jeukendrup*
M: +31 (0) 612063127
E: bart@jeukendrup.nl
LinkedIn <https://nl.linkedin.com/in/bartjeukendrup> | PGP
<http://www.bartj.com/email/>

2015-05-01 13:41 GMT+02:00 Jerôme Bakker <notifications@github.com>:

**Comment by jdalsem** [comment]
Please provide a new correct PR as discussed in other composer related PR's

**Pull request closed without merging**